### PR TITLE
[FIX][8.0][base_partner_merge] Merging two partners with same follower fails no more. Fixes #179

### LINKED ISSUE #179: [base_partner_merge] merging two partners both followed by the same partner fails.
To reproduce with demo data, make two copies of Agrolait. Both will be followed by your user's partner.
Attempt to merge them raises a constraint error saying: "Error, a partner cannot follow twice the same object."

### PR DESCRIPTION
**Pull request by charbeljc**

### CODE REVIEW

**Comment by dreispt** [comment]
You should use a more meaningful commit message.
For example: `Merging two partners with same follower failed. Fixes #179`

**Comment by guewen** [comment]
:+1:

**Comment by charbeljc** [comment]
Hi @dreispt I restated this PR following your advice, but travis seams stale and I have no view on the build queue, could you help me on this ?

File: base_partner_merge/README.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+`here <https://github.com/OCA/partner-contact/issues/new?body=module:%20base_partner_merge%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Firstname Lastname <email.address@example.org>
```
Comment (damdam-s) [inline base_partner_merge/README.rst]: You could add yourself here

File: base_partner_merge/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Base Partner Merge
+==================
+
+
+This module implements merging of multiple partners
+depending on thier similarity
```
Comment (damdam-s) [inline base_partner_merge/README.rst]: s/thier/their/

**Comment by damdam-s** [comment]
some remarks else :+1:

**Comment by rafaelbn** [comment]
Tested in runbot :+1: 
Please review comment above, thanks.

**Comment by rafaelbn** [comment]
Please check https://github.com/OCA/partner-contact/issues/196 as @guewen said

**Comment by charbeljc** [comment]
Hum, I didn't notice the duplication of this module from crm. Thanks @rafaelbn & @guewen for pointing this.  How should we handle this situation ? we currently have PR #182 and #183 for the duplicated OCA module, should I resubmit them on the base crm package ?

File: base_partner_merge/__init__.py [hunk 3]
```diff
@@ -1 +1,2 @@
-from . import base_partner_merge
+# -*- coding: utf-8 -*-
+from . import base_partner_merge  # NOQA
```
Comment (moylop260) [inline base_partner_merge/__init__.py:2]: `NOQA`?

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**